### DISCUSSION TITLE
Лучшие настройки/ Best settings

### DISCUSSION THREAD

**Discussion by Thxoode** [shamhi/HamsterKombatBot#2438] [General]
Есть ли какие то настройки,которые уменьшат шанс получить карточку читера? / Are there any settings that will reduce the chance of getting a cheating card?

**Comment by Insomnia731** [comment]
Нужна стата. Было всего 2 акка, запустил их неделю назад, там миллионов по 7 монет и только вот уехали в баню. Я думал обращают внимание на какие-то запредельные цифры по всем достижениям, а вот нифига. И мало того даже если айпи пересекались, так акка-то всего 2.
Неясно короче.

Нужна инфа от тех людей кто юзал скрипты и кому бан не прилетел, в чём секрет.

**Comment by Insomnia731** [comment]
И да ходят слухи что за фарм ключей в бан уехали. Неясно насколько правда.
А вообще мне кажется основное фингерпринты, если устройство липовое по ним вернее всего можно спалить, хотя я не думал что при таком объеме юзеров это всё будут так перепроверять. Всё остальное, ну айпи может быть любой, в наше-то время впн у всех, хедеры тоже чепуха это всё легко меняется и по ним палить такое себе.

**Comment by Danielz1452** [comment]
Точно не по айпи, улетели 50 акков, каждый на уникальной проксе стоял. Вероятнее всего фарм ключей, но! Тех кто вручную вводил ключи с генератора разбанили сегодня ночью, полагаю банили по другой причине.

**Comment by RecursionHs** [comment]
This is an interesting topic. I think it's still behavior and can be optimized from two aspects.
1.Increase the overall start time and end time, mainly to keep all accounts from starting at the same time (now it can be done through startup delay), mainly for the end time, I think if there is a delayed end, it would be better, normal people cannot be online 24 hours a day. ;
2.The acquisition of keys, it is recommended to add a configuration of a random number for obtaining the number of keys, instead of obtaining all game keys every time, only need to obtain the keys of the specified random number. This random number is generated only once a day and can be placed in the profile file.
